config/routes.rb

```diff
@@ -3,4 +3,5 @@
 
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   root 'welcome#index'
+  resources :images
```

Comment:
Let's limit the routes created here just to the ones we use (`show`, `new`, and `create`): http://guides.rubyonrails.org/routing.html#restricting-the-routes-created